### DISCUSSION TITLE
使用VcGraphicsPolyline画实体之间的连线问题

### DISCUSSION THREAD

**Discussion by shaopGao** [zouyaoji/vue-cesium#521] [Q&A]
使用VcGraphicsPolyline画实体之间的连线第一时间不会显示出来，需要点击选中一个实体才会刷新
![Uploading 1.png…]()
![Uploading 2.png…]()

**Comment by shaopGao** [comment]
<img width="356" alt="1" src="https://github.com/zouyaoji/vue-cesium/assets/115469092/3477a6c4-6779-40b0-965e-4927e2a01446">

**Comment by shaopGao** [comment]
<img width="339" alt="2" src="https://github.com/zouyaoji/vue-cesium/assets/115469092/d377f868-b07a-48d2-9cc3-ebd7ad12a8cc">

**Comment by zouyaoji** [comment]
不清楚咋回事，建议弄个在线复现例子瞅瞅看看。
